spellchecker/symspell.py

```diff
@@ -137,29 +161,20 @@ def create_dictionary_entry(self, key, count):
                 self._deletes[delete_hash] = [key]
         return True
 
-    def load_dictionary(self, corpus, term_index, count_index):
-        """Load multiple dictionary entries from a file of word/frequency
+    def load_dictionary(self, word_iterator: Iterable[tuple]) -> None:
```

Comment:
Whenever a function returns nothing you can omit the typehint return value as well as the Returns part of the docstring